### ISSUE TITLE
Live Tile Content should be different on back side and front side of Tile

### ISSUE THREAD

**Issue by hakim-ali-confiz** [timheuer/callisto#221]
Hey,

I'm using live tile in mine project, and i really appreciate for your great work,

i have work on one Enhancement of Live Tile, By default apps get a Flip Tile which has a front (usually displaying an icon and summery of tile) and back (usually displaying text) and periodically flips between them. 

Please let me know how can i contribute to your project.

Regards,
Hakim Ali

**Comment by dotMorten** [comment]
Could this be a separate control with two or more contents? Then you could just put the LiveTile inside the front part of the FlipTile.
The way to contribute is to fork the project, add the control to your fork and submit a pull-request. After the code has been reviewed and tested, it can be accepted back into the callisto repo.

**Comment by hakim-ali-confiz** [comment]
Commit 65ccf4e5
hakim-ali-confiz
hakim.ali@confiz.com
11/4/2013 8:05:13 PM
Parent: e4367080

Add WindowLiveTile Control into Callisto Controls and in samplePage.

To Do List in this commit
1- Add a new folder "WindowLiveTile" into Controls repository.

2- Add a new class "WindowLiveTile.cs" into folder "WindowLiveTile" folder.

3- Add Sample Page for WindowLiveTile in SamplePages with Name "WindowLiveTileSample" and add sample grid into this.

4- Add refernce of sample page into Main Page

Kindly review my commit, and let me know if i have made some modification. 

Basically i contribute a live tile(With some modification with i have mentions in first comment), but you have already LiveTile that's why i named it WindowLiveTile. You can modify accordingly to your choice.

I'm waiting for your positive response.

Regards,
Hakim Ali

**Comment by timheuer** [comment]
@hakim-ali-confiz I think we will pull the template selector fix in instead of a whole new control

**Comment by hakim-ali-confiz** [comment]
ok. thanks timheuer. Let me know if I can be of any other assistance in this.
